resources/opt/sonatype/nexus/resources/nexusSetupCleanupPolicies.groovy

```diff
@@ -0,0 +1,40 @@
+import org.sonatype.nexus.cleanup.storage.CleanupPolicyStorage
+import groovy.json.JsonSlurper
+
+// get parameters from payload JSON file
+def configurationParameters = new JsonSlurper().parseText(args)
+
+/**
+ * this script is called in startup.sh and creates a cleanup
+ * @return
+ */
+def createMavenSnapshotCleanupPolicy(configurationParameters) {
+    try {
+        def policyStorage = container.lookup(CleanupPolicyStorage.class.getName())
+
+        def cleanupPolicy = policyStorage.newCleanupPolicy()
+        cleanupPolicy.setName(configurationParameters.name)
+        cleanupPolicy.setNotes(configurationParameters.notes)
+        cleanupPolicy.setMode(configurationParameters.mode)
+        cleanupPolicy.setFormat(configurationParameters.format)
+
+        def criteriaMap = [:]
+        criteriaMap.put("regex", configurationParameters.criteria.regex)
+        criteriaMap.put("criteriaReleaseType", configurationParameters.criteria.criteriaReleaseType)
+        criteriaMap.put("criteriaLastBlobUpdated", configurationParameters.criteria.criteriaLastBlobUpdated)
```

Comment:
this does not work likewise, but here need some calculations to be made in order take effect. Like so:
```suggestion
        criteriaMap.put("lastBlobUpdated", configurationParameters.criteria.criteriaLastBlobUpdated * yourCalculationHere)
```

- [ ] done.

Reply:
Added helper methods to convert a days to a seconds value.